json-server/config/routes.json

```diff
@@ -6,6 +6,7 @@
     "/:personId/notes": "/notes",
     "/users/current": "/users",
     "/deputies/:deputyId/ecm": "/ecm",
+    "/successes/deputies/:id/contacts": "/contacts",
```

Comment:
This isn't really how json-server should be used. Treat it as if it were Sirius i.e. a real database that you access via endpoints, rather than just a repository of data for tests. `db.js` should have a collection called "contacts" with deputy ids, which you can then re-route to like `"/contacts?deputyId=:id"`.

This also doesn't appear to be the correct route, as I'm not sure what `/successes` refers to.